src/leaflet.filelayer.js

```diff
@@ -105,6 +105,24 @@
             return reader;
         },
 
+        loadMultiple: function (files, ext) {
+            var reader = false;
+            if (files[0]) {
+              files = Array.prototype.slice.apply(files);
+              var loadOne = L.Util.bind(function () {
+                reader = this.load(files.shift(), ext, reader);
+                if (files.length > 0) {
+                    setTimeout(loadOne, 25);
```

Comment:
Unfortunately, this comes from before I joined the team maintaining the project so I can't tell if it's still relevant. :(
My first intuition would be it was to prevent some old browsers from struggling at loading multiple files at once. We can go without it from now on I think, and eventually, we'll reintroduce it if needed (With a meaningful comment this time x) )